e2e/config.go

```diff
@@ -3,44 +3,71 @@ package e2e
 import (
 	"flag"
 	"fmt"
+	"os"
 
-	deploymanager "github.com/red-hat-storage/lvm-operator/pkg/deploymanager"
+	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
+	operatorv1 "github.com/operator-framework/api/pkg/operators/v1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	lvmv1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// TestNamespace is the namespace we run all the tests in.
-const TestNamespace = "lvm-endtoendtest"
-const InstallNamespace = "openshift-storage"
-
-var lvmOperatorInstall bool
-var lvmOperatorUninstall bool
-
-// LVMCatalogSourceImage is the LVM CatalogSource container image to use in the deployment
-var LvmCatalogSourceImage string
-
-// LvmSubscriptionChannel is the name of the lvm subscription channel
-var LvmSubscriptionChannel string
-
-// DiskInstall indicates whether disks are needed to be installed.
-var DiskInstall bool
-
-// DeployManager is the suite global DeployManager
-var DeployManagerObj *deploymanager.DeployManager
-
-// SuiteFailed indicates whether any test in the current suite has failed
-var SuiteFailed = false
+const (
+	testNamespace    = "lvm-endtoendtest"
```

Comment:
Please add comments describing the constants and variables.

Reply:
Updated